src/Commands/ImportDump.php

```diff
@@ -234,10 +234,20 @@ public function handle()
             }
         }
 
-        if ($importFilePath && ($optionForce || $this->confirm(sprintf('Are you sure that you want to import the dump at %s?', $disk->path($importFilePath))))) {
+        $this->info(sprintf('Dump: "%s"', $disk->path($importFilePath)));
+        $this->info(sprintf('Database: "%s"', $protector->getDatabaseName()));
```

Comment:
What was so bad about having the database as part of the confirm?
This adds so much unnecessary visual noise...

Reply:
sure we can do that. it just seemed a bit much for one line.

Reply:
can be reduced to just asking for the database name in the question

Reply:
done.